src/shared/util.js

```diff
@@ -454,6 +454,7 @@ function bytesToString(bytes) {
 }
 
 function stringToBytes(str) {
+  assert(str, 'Invalid or empty string passed to stringToBytes');
```

Comment:
`typeof str` is nice, since it actually checks that it's a string. But if the result of micro benchmarks is to be trusted, the performance is unfortunately a bit varied in different browsers: http://jsperf.com/typeof-vs-undefined-null. Just do what you want here :-)